.circleci/config.yml

```diff
@@ -44,7 +44,7 @@ jobs:
             - src/deps
             - src/_build
       - run:
-          command: sh .circleci/ci_slack.sh failure
+          command: if [ "$CIRCLE_BRANCH" = "master" ]; then sh .circleci/ci_slack.sh failure; fi
```

Comment:
Maybe do this in `ci_slack.sh`? Just `exit` will do.

Reply:
It feels weird to have this master condition in `ci_slack.sh`? Feels like you already have all the usage conditionals in this yml file...